js/location.js

```diff
@@ -0,0 +1,88 @@
+(function(module) {
+  function userLocation(opts) {
+    Object.keys(opts).forEach(function(e, index, keys) {
+      this[e] = opts[e];
+    }, this);
+  }
+
+  userLocation.all = [];
+
+  userLocation.createTable = function(callback) {
+    webDB.execute(
+      'CREATE TABLE IF NOT EXISTS locations (' +
+        'id INTEGER PRIMARY KEY, ' +
+        'name VARCHAR(255) NOT NULL, ' +
+        'address VARCHAR(255) NOT NULL, ' +
+        'city VARCHAR(60) NOT NULL, ' +
+        'state VARCHAR(2) NOT NULL, '+
+        'zip VARCHAR(10) NOT NULL);',
```

Comment:
Maybe we should use Integer here and limit to 5 digit zip (because who really knows their full zipcode?)